src/components/chat/comment-card.tsx

```diff
@@ -18,8 +19,9 @@ interface IProps {
 export const CommentCard: React.FC<IProps> = ({ activeNavTab, user, postedComments, onPostComment }) => {
    const renderThreadHeader = () => {
     const teacherInitial = user?.name.charAt(0);
+    const commentCardHeaderClass = classNames("comment-card-header", "comment-select");
```

Comment:
I'm fine with using `classNames()` here if you prefer but given that there are just two string constants, you don't really need it in this case.